service/github/github.go

```diff
@@ -250,3 +253,44 @@ func listAllPullRequestsComments(ctx context.Context, cli *github.Client,
 	}
 	return append(comments, restComments...), nil
 }
+
+func buildBody(c *reviewdog.Comment) string {
+	cbody := commentutil.CommentBody(c)
+	if suggestion := buildSuggestions(c); suggestion != "" {
+		cbody += "\n" + suggestion
+	}
+	return cbody
+}
+
+func buildSuggestions(c *reviewdog.Comment) string {
+	var sb strings.Builder
+	for _, s := range c.Result.Diagnostic.GetSuggestions() {
+		txt, err := buildSingleSuggestion(c, s)
+		if err != nil {
+			log.Printf("reviewdog: %v", err)
+			continue
+		}
+		sb.WriteString(txt)
+		sb.WriteString("\n")
+	}
+	return sb.String()
+}
+
+func buildSingleSuggestion(c *reviewdog.Comment, s *rdf.Suggestion) (string, error) {
+	start := s.GetRange().GetStart()
+	if start.GetLine() != c.Result.Diagnostic.GetLocation().GetRange().GetStart().GetLine() {
+		return "", errors.New("Diagnostic and Suggestion lines must be the same.")
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
error strings should not be capitalized or end with punctuation or a newline